src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(x => x.sex === 'm');
+  const menFilter = century
```

Comment:
You need more reliable check here because century may theoretically have valid falsy value (0).
```suggestion
  const menFilter = century !== undefined
```